lib/handlers/events/delivery/goals/SkipDownstreamGoalsOnGoalFailure.ts

```diff
@@ -57,20 +57,25 @@ export class SkipDownstreamGoalsOnGoalFailure implements HandleEvent<OnAnyFailed
             .filter(g => g.state === "planned");
 
         let failedGoalState;
+        let failedGoalDescription;
         switch (failedGoal.state) {
             case SdmGoalState.failure:
-                failedGoalState = "failed";
+                failedGoalDescription = "failed";
+                failedGoalState = SdmGoalState.skipped;
                 break;
             case SdmGoalState.stopped:
-                failedGoalState = "stopped goals";
+                failedGoalDescription = "stopped goals";
+                failedGoalState = SdmGoalState.skipped;
```

Comment:
Should this be `SdmGoalState.stopped`?

Reply:
No, we are skipping because of a stopped goal. That is correct.